WalletWasabi/JsonConverters/Bitcoin/CoinJsonConverter.cs.cs

```diff
@@ -0,0 +1,38 @@
+using NBitcoin;
+using Newtonsoft.Json;
+
+namespace WalletWasabi.JsonConverters.Bitcoin;
+
+public class CoinJsonConverter : JsonConverter<Coin>
+{
+	/// <inheritdoc />
+	public override Coin? ReadJson(JsonReader reader, Type objectType, Coin? existingValue, bool hasExistingValue, JsonSerializer serializer)
+	{
+		var coin = serializer.Deserialize<SerializableCoin>(reader);
```

Comment:
Maybe

```suggestion
		var coin = serializer.Deserialize<SerializableCoin>(reader) ?? throw new JsonException();
```
So the warning below this line disappears.